PROJECT_PROPOSAL.md

```diff
@@ -0,0 +1,95 @@
+
+The public has no idea that any of this is going on. In fact they have little understanding of what mathematicians do at all. There *is* public interest in mathematics, as demonstrated by the popularity of events like the "Summers of Math Exposition" on YouTube. However, most of the infrastructure of contemporary mathematics remains out-of-reach except to experts.
+
+This might be surprising to people working in these fields. Contemporary algebra, and "higher category theory" especially, tend to build on simple and intuitive spatial metaphors. For example:
+- Rearranging trees (traditional algebra)
+- Gluing together triangles (commutative diagrams)
+- Sliding around beads on tangled wires (string diagrams)
+And so on. How could anybody not understand these ideas?
+
+Part of the issue is that the literature is dense and abstract. There's a saying:
+> "Trying to learn math by reading only published papers is like trying to learn chess by reading only chess notation."
+Published proofs tend to convey exactly enough information to convince a *specialist* that an argument works. A non-specialist, however, will have absolutely no idea what's going on.
+
+This applies twice over to computer-checked proofs. Proof systems are targeted towards expert users, who are *already familiar* with various complicated rulesets (software development tools, dependent type systems, higher-order unification, etc.) A proof written with such an assistant is very convincing *to the person who wrote it*, and maybe her friends, but to non-experts it is a black box. 
+
+Other mathematicians may be convinced *that* the proof is correct, simply by its being written in a proof assistant. Surprisingly, though, looking at its source code will not tell them much about how it works, at least without in-depth study. This makes it harder to establish trust within the mathematical community. Results that have only been checked by computer, such as the Four-Color theorem, are widely mistrusted, and much effort has been made to find fully human-checkable proofs of the same.
+
+To summarize: a lot of very nice formalism is going on. However, most of it is inaccessible to most everybody.
+
+It would be nice to think that exposition alone could solve this problem. However, for people who actually need to use mathematics, exposition alone is not enough. To truly communicate a mathematical result, not only the *what* must be communicated, but also the *how*, and at least some of the *why*. Someone studying mathematics can't just memorize it by rote: they have enact it themselves. To quote a [physics educator](https://worrydream.com/refs/Hestenes_2002_-_Reforming_the_Mathematical_Language_of_Physics.pdf):
```

Comment:
I know Hestenes can raise eyebrows so it might be good to find somebody else saying the same thing. I'm only really interested in what he has to say about teaching.